### PR TITLE
Fix incorrect computation of excited state population

### PR DESCRIPTION
**Pull request by itoko**
### Summary
Fix a bug in computation of the excited population from qubit frequency and temperature that can be used in noise model construction. 

### Details and comments
As reported in #1638, there was an incorrect exponential factor of 2 and hence the excited population was incorrect (almost squared). This fix will affect users who generate a noise model using `NoiseModel.from_backend` with `temperature` option while it won't affect users of the default noise model that assumes no excitation (temperature=0).

### CODE REVIEW

**Review by hhorii — DISMISSED** [review]

File: releasenotes/notes/fix-temperature-a9c51c4599af3a49.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+fixes:
+  - |
+    Fixed a bug in computation of the excited population from qubit frequency and temperature
```
Comment (chriseclectic) [inline releasenotes/notes/fix-temperature-a9c51c4599af3a49.yaml:4]: I would reword to stress the error condition first eg:
> Fixed a bug in `NoiseModel.from_backend` where using the `temperature` kwarg with a non-default value would incorrectly compute the excited state population for the specified temperature. Previously, there was an additional factor of 2 in the Boltzman distribution calculation leading to an incorrect smaller value for the excited state population.

**Review by chriseclectic — COMMENTED** [review]
Looks good. Suggestion for wording of release note

**Review by hhorii — APPROVED** [review]
